lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -155,21 +169,21 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
         finishButton.tintColor = LucienTheme.dark
     }
 
-    // MARK: - IBOutlet Methods
+    @objc private func releaseDateEditingChanged(_ textField: UITextField) {
+        guard let text = textField.text else {
+            return
```

Comment:
This can be on the same line as the `guard`. Guard statements should be one liners unless the return is doing something more than just returning